data.sql

```diff
@@ -4,3 +4,10 @@ INSERT INTO animals VALUES(1,'Agumon','2020-02-03',0,'1',10.23);
 INSERT INTO animals VALUES(2,'Gabumon','2018-11-15',2,'1',8);
 INSERT INTO animals VALUES(3,'Pikachu','2021-01-07',1,'0',8);
 INSERT INTO animals VALUES(4,'Agumon','2017-05-12',5,'1',11);
+INSERT INTO animals VALUES(5,'Charmander','2020-02-08',0,'0',11);
+INSERT INTO animals VALUES(6,'Plantmon','2021-11-15',2,'1',5.7);
+INSERT INTO animals VALUES(7,'Squirtle','1993-04-02',3,'0',12.13);
+INSERT INTO animals VALUES(8,'Angemon','2005-06-12',1,'1',45);
```

Comment:
- [ ]  Please kindly add the `-` to your weights is the negative numbers, not positive number the values are `-11.0, -5.7, -12.13, -45.0` 🙃 🙃

Reply:
Thank you @cynthiainga